pages/Page.vue

```diff
@@ -1,7 +1,158 @@
+})
+const page_data_cookie=useCookie('cv');
+const page_data= computed (() => JSON.parse(page_data_cookie.value || "{}"))
+const router=useRoute()
+const id= computed(() => {router.params.id});
+
+	/* update the progress bar based on donation amount !!!!!
+	const progress=document.querySelector('.progress') || "{}";
+	progress.style.width=progress.getAttribute('donated-amount') + "%";
+	progress.style.opacity=1;
+    */
+	
+}
+
+</script>
 
 <template lang="pug">
-div This is where a page will be rendered. Use route or query parameters to identify which page needs to be loaded.
+doctype html 
+head 
+	link(rel='preconnect' href='https://fonts.googleapis.com/')
+	link(rel='preconnect' href='https://fonts.gstatic.com/' crossorigin='')
+	link(href='https://fonts.googleapis.com/css2?family=Poppins:ital@0;1&display=swap' rel='stylesheet')
+	link(href='https://fonts.googleapis.com/css2?family=Outfit:wght@600&display=swap' rel='stylesheet')
+html
+	head
+		
+		link(rel='stylesheet' href='/stylesheets/main.css')
+	.bg-blue.w-screen(class="h-[80px]")
+	img.bg-orange-300.-mt-16.mx-auto(class="w-[122px] h-[122px] rounded-[8px]" src=profile)
+	.text-gray-dark.font-poppins.text-2xl.text-center.font-semibold(style="text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);") {{data.page_name}}
+	.row.text-center
+		.text-gray-dark.font-poppins.text-md.inline-block.text-center {{data.day_of_birth}}
+		.text-gray-dark.font-poppins.text-md.inline-block.whitespace-pre  - 
+		.text-gray-dark.font-poppins.text-md.inline-block {{data.day_of_passing}}
+	.row.p-3
+		NuxtLink.mx-16.p-3.px-6.pt-2.text-white.bg-orange-400.font-poppins(style="font-weight: 700; border-radius: 32px;" to='/pages') Back
+	.container.bg-gray-light.mx-auto.h-auto.font-poppins(class="w-11/12 rounded-[8px]" style="box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);")
+		//ul.inline-flex.py-1.px-1.w-full(id="tabs")
+			li.bg-white.text-xs.text-center.px-auto.py-2(class="rounded-[8px] w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" id="default-tab" href="#obituary") Obituary
+			li.text-xs.text-center.px-auto.py-2(class="w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" href="#services") Services
+			li.text-xs.text-center.px-auto.py-2(class="w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" href="#donations") Donations
+			li.text-xs.text-center.px-auto.py-2(class="w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" href="#media") Media
+
+		.div(style="background: #F8F8F8;" id="tab-contents")
+			.div.px-8.py-4(style="color: #6E6E6E; font-weight: 500; font-size: 14px; line-height: 28px; letter-spacing: -0.078px; word-break: break-word;" id="obituary") {{data.obituary}} Obituary 
+			.div.p-4(id="services")
+				if visitation_date
```

Comment:
Update template usage - this needs to use Vue's interpolation and [conditional rendering](https://vuejs.org/guide/essentials/conditional.html)